greylist_db.py

```
@@ -0,0 +1,76 @@
+        await db.execute("""
+        INSERT INTO greylist(email, mx_host, last_try, next_try, tries)
+        VALUES (?, ?, ?, ?, 1)
+        ON CONFLICT(email) DO UPDATE SET
+            mx_host = excluded.mx_host,
+            last_try = excluded.last_try,
+            next_try = excluded.next_try,
+            tries = greylist.tries + 1;
+        """, (email, mx_host, now, next_try))
+        await db.commit()
+
+def upsert_greylist_sync(email: str, mx_host: str, retry_delay: int, tries: int = 1):
```

Ruff (0.12.2)

Unused function argument: `tries`

(ARG001)